app/models/resource_pool.rb

```diff
@@ -121,6 +121,14 @@ def parent_resource_pool
     parent(:of_type => 'ResourcePool')
   end
 
+  def parent_resource_pool_cloud
+    parent(:of_type => 'ManageIQ::Providers::CloudManager::ResourcePool')
+  end
+
+  def parent_resource_pool_infra
+    parent(:of_type => 'ManageIQ::Providers::InfraManager::ResourcePool')
+  end
+
```

Comment:
TODO I'm not sure that we need these

Reply:
Yeah, the `parent_resource_pool` does what I need, hence I'm removing these extras.